lib/tasks/fetch.rake

```diff
@@ -6,6 +6,18 @@ task fetch_all_rss: :environment do
   RssReader.get_all_articles(force: false) # don't force fetch. Fetch "random" subset instead of all of them.
 end
 
+task fetch_rss_reader: :environment do
+  Rails.application.eager_load!
```

Comment:
Is it necessary to eager load all of Rails here or could you get away with `require_dependency`?

Reply:
Good question, don't know. I just copy and pasted the previous one. I'll try and let you know :)